pkgs/applications/science/misc/gephi/default.nix

```diff
@@ -52,7 +52,7 @@ stdenv.mkDerivation {
     cp ${javaPackages.jogl_2_3_2}/share/java/jogl*.jar $out/gephi/modules/ext/org.gephi.visualization/org-jogamp-jogl/
     cp ${javaPackages.jogl_2_3_2}/share/java/glue*.jar $out/gephi/modules/ext/org.gephi.visualization/org-jogamp-gluegen/
 
-    echo "jdkhome=${jdk}" >> $out/etc/gephi.conf
+    printf "\njdkhome=${jdk}" >> $out/etc/gephi.conf
```

Comment:
gephi would fail with `Unrecognized option: --add-opens=java.base/java.net=ALL-UNNAMED`, as this was being added to the `extra_clusters` line instead of a separate one